packages/main/src/mainWindow.ts

```diff
@@ -72,18 +73,20 @@ async function createWindow(): Promise<BrowserWindow> {
   }
 
   const browserWindow = new BrowserWindow(browserWindowConstructorOptions);
+
   const { getCursorScreenPoint, getDisplayNearestPoint } = screen;
   const workArea = getDisplayNearestPoint(getCursorScreenPoint()).workArea;
 
   const x = Math.round(workArea.width / 2 - INITIAL_APP_WIDTH / 2 + workArea.x);
   const y = Math.round(workArea.height / 2 - INITIAL_APP_HEIGHT / 2);
 
-  browserWindow.setBounds({
+  const initialBounds: Rectangle = {
     x: x,
     y: y,
-    width: browserWindowConstructorOptions.width,
-    height: browserWindowConstructorOptions.height,
-  });
+    width: INITIAL_APP_WIDTH,
```

Comment:
Not sure to understand this change

Reply:
it's a shortcut as we were giving INITIAL_APP_WIDTH to browserWindowConstructorOptions but I can withdraw the change as at some point the code was elsewhere but now it went back there so I have access to browserWindowConstructorOptions

Reply:
ok now I recall, it's because browserWindowConstructorOptions.width return a number or undefined and then we don't want to have an undefined value